.github/linters/.python-lint

```diff
@@ -168,11 +168,10 @@ disable=abstract-method,
       range-builtin-not-iterating,
       raw_input-builtin,
       raw-checker-failed,
-      rdiv-method,
```

Comment:
and is this the one that you actually working on?

Reply:
I'm not working on that one. I was working on the warning redefined-outer-name.

Reply:
I'll start working on that warning right now.